instaclustr/structs.go

```diff
@@ -28,8 +28,11 @@ type BundleOptions struct {
 }
 
 type ClusterProvider struct {
-	Name        *string `json:"name" mapstructure:"name"`
-	AccountName *string `json:"accountName, omitempty" mapstructure:"account_name"`
+	Name              *string `json:"name" mapstructure:"name"`
+	AccountName       *string `json:"accountName, omitempty" mapstructure:"account_name"`
+	Tags              *string `json:"tags,omitempty" mapstructure:"tags"`
+	ResourceGroup     *string `json:"resourceGroup,omitempty" mapstructure:"resourceGroup"`
```

Comment:
disk_encryption_key and resource_group in schema doesn't work because the mapstructure variables are incorrect here

Reply:
To clarify, the new mapstructure variable names used here should be snake cased instead of camel cased, like the other properties of the struct